tests/bluetooth/bsim_bt/bsim_test_audio/src/mcc_test.c

```diff
@@ -37,6 +37,7 @@ static uint64_t g_next_track_object_id;
 static uint64_t g_current_group_object_id;
 static uint64_t g_parent_group_object_id;
 static uint64_t g_search_results_object_id;
+static uint64_t g_tmp_object_id;
```

Comment:
Can't this be a local variable? Doesn't look like it needs to be global.

Reply:
It can be local.  (Will be "half-global" anyway, as the test_main function is so large ...)
Changed.